mchf-eclipse/hardware/uhsdr_board.h

```diff
@@ -1021,6 +1024,7 @@ typedef struct TransceiverState
 
 	keyer_mode_t keyer_mode; // disable/enable keyer mode for F1-F5 buttons
 	bool buffered_tx; // disable/enable buffered sending for CW and digital modes
+	bool stereo_enable; // enable/disable stereo demodulation (only in special hardware, NOT in mcHF)
```

Comment:
Why not ifdef it not usable on all hardware?
Allows us to clearly identify all related places for hardware specific features. And the compiler will tell you where you make references to stereo_enable in "mcHF" code, where it is not supported at all.

Reply:
Done.